core/frontend-stubs-testing/src/main/java/org/eclipse/sw360/antenna/frontend/testing/testProjects/ExampleTestProject.java

```diff
@@ -84,6 +84,8 @@ public String getExpectedToolConfigurationProductVersion() {
         analyzers.add(mkWorkflowStep("CSV Analyzer", "org.eclipse.sw360.antenna.workflow.analyzers.CsvAnalyzer",
                 "base.dir", this.projectRoot.toString(),
                 "file.path", "src/dependencies.csv"));
+        analyzers.add(mkWorkflowStep("ORT Result Analyzer", "org.eclipse.sw360.antenna.ort.workflow.analyzers.OrtResultAnalyzer",
+                "file.path", "src/analyzer-result.yml"));
```

Comment:
I think you also need to add the `src/analyzer-result.yml` to https://github.com/eclipse/antenna/blob/17f055e7728ece7a0cbcc71189394b11962cb3f8/core/frontend-stubs-testing/src/main/java/org/eclipse/sw360/antenna/frontend/testing/testProjects/ExampleTestProject.java#L40-L46

Reply:
Isn't this just for policies?

Reply:
Yes, the functions `getOtherFilesToCopy` and `getOutOfProjectFilesToCopy` can now be merged, but you still need to add the new file, so that it gets copied to the temporary folder if a test wants to set up the Example Project.

The result could maybe look like:
```java
    @Override
    public List<String> getOtherFilesToCopy() {
        return Stream.of("src/reportData.json", "src/dependencies.csv",
                "src/example-policies/rules/DummyRule.drl", "src/example-policies/policies.properties", "src/example-policies/policies.xml",
                "src/analyzer-result.yml")
                .collect(Collectors.toList());
    }

    @Override
    public List<String> getOutOfProjectFilesToCopy() {
        return Stream.of(
                "./src/example-policies/rules/DummyRule.drl",
                "./src/example-policies/policies.properties",
                "./src/example-policies/policies.xml")
                .collect(Collectors.toList());
    }
```